packages/vue/src/components/molecules/SfSteps/SfSteps.js

```diff
@@ -45,6 +45,11 @@ export default {
       default: true
     }
   },
+  data() {
+    return {
+      progressWidth: `${100 / this.steps.length}%`
```

Comment:
This should be a computed no ?

Reply:
I originally had it as a computed but then I thought the amount of steps shouldn't change so it was unnecessary but maybe it should be in case steps are added/removed on the fly.

Reply:
Fixed